<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>621</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="View_order_button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>410</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Просмотреть заказ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_order_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить заказ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>5</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_dish_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Traditional Nushu</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Добавить меню</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="Menu_tree_widget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>481</width>
      <height>431</height>
     </rect>
    </property>
    <property name="uniformRowHeights">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>Меню</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="Sort_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>130</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>883</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="Menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="Menu_action"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="Order_list">
    <property name="title">
     <string>Список заказов</string>
    </property>
    <addaction name="Order_action"/>
   </widget>
   <addaction name="Menu"/>
   <addaction name="Order_list"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Menu_action">
   <property name="text">
    <string>Просмотр текущего меню</string>
   </property>
  </action>
  <action name="Order_action">
   <property name="text">
    <string>Просмотреть список</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
